### DISCUSSION TITLE
I Lost my Mesh Toke :(

### DISCUSSION THREAD

**Discussion by MestrePadawan** [amidaware/tacticalrmm#1284] [General]
Hello guys,
I lost myself my MESH TOKEN on (Settings > Global Settings > Mesh Central > Mesh Token.

This made the agents Install dont download mesh agent the RMM.

Is the a solution for this? Can I reset my MESH TOKEN on Settings > Global Settings > Mesh Central > Mesh Token??

Please help me  :(

**Comment by dinger1986** [comment]
https://docs.tacticalrmm.com/troubleshooting/#need-to-recover-your-mesh-token

Should get you sorted out 😃

**Reply by MestrePadawan** [reply]
I will see.....if i made, i come back for. Thank YOU

**Reply by MestrePadawan** [reply]
Owwww I LOVE YOU.
Its working :)

tks e sorry for the english.
